pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1311,6 +1342,57 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundThriftListenerOptsForP
 	}}
 }
 
+func (configgen *ConfigGeneratorImpl) buildSidecarOutboundDubboListenerOptsForPortOrUDS(listenerMapKey *string,
```

Comment:
You dont need all this.. Literally just add the dubbo filter into the networkfilters.go and you should be good to go.

Reply:
Unlike mysql/mangodb, Dubbo filter handles the routing by itself instead of passing it through to a tcp proxy filter.